Formula/wget2.rb

```diff
@@ -59,7 +59,7 @@ def install
                           "--without-libmicrohttpd"
     if build.head?
       system "make"
-    elsif os.linux?
+    elsif OS.linux?
       system "make", "LIBS+=-lgpgme"
```

Comment:
I don't know why, but explicitly linking `gpgme` seemed to work. I will try without this to build, and if `-lgpgme` is indeed required I will add it back later.